common/src/comments.rs

```diff
@@ -107,7 +111,8 @@ where
     delegate!();
 }
 
-pub type SingleThreadedCommentsMap = Rc<RefCell<FxHashMap<BytePos, Vec<Comment>>>>;
+pub type SingleThreadedCommentsMapInner = FxHashMap<BytePos, Vec<Comment>>;
+pub type SingleThreadedCommentsMap = Rc<RefCell<SingleThreadedCommentsMapInner>>;
```

Comment:
I'm not sure about this naming, but I didn't want to cause a breaking change.

Reply:
It seems ok to me.